pkg/log/log.go

```diff
@@ -224,7 +228,7 @@ func Container(ctx context.Context, pod *corev1.Pod, name string, output chan<-
 			}
 		} else if err == nil {
 			pod, err := options.PodClient.Pods(pod.Namespace).Get(ctx, pod.Name, metav1.GetOptions{})
-			if err == nil && (!pod.DeletionTimestamp.IsZero() || pod.Status.Phase == corev1.PodSucceeded || pod.Status.Phase == corev1.PodFailed) {
+			if err == nil && (objectDeleted(pod) || pod.Status.Phase == corev1.PodSucceeded || pod.Status.Phase == corev1.PodFailed) {
```

Comment:
I'm not sure if it is possible to get an object that has been deleted and has no finalizers, but it's probably fine here?